addons/web/controllers/pivot.py

```diff
@@ -102,9 +102,10 @@ def export_xlsx(self, data, token):
 
         workbook.close()
         xlsx_data = output.getvalue()
+        filename = 'Pivot of ' + jdata['title'] + '(' + jdata['model'] + ').xlsx'
```

Comment:
I imagine this should be translatable, using placeholders that translators can understand ;-)

e.g. something like:

```py
filename = _("Pivot of %(screen_title)s (%(doctype)s).xlsx",
             screen_title=jdata['title'], 
             doctype=jdata['model'])
```